### PR TITLE
Feat/recv2- Recording functionality

### PR DESCRIPTION
**Pull request by AdityaBITMESRA**
Updated the create_new_vispy_canvas code to add video recording functionality

### CODE REVIEW

**Comment by AdityaBITMESRA** [comment]
Hi sir @sanjayankur31. Can you take a look at this?

File: pyneuroml/plot/PlotMorphologyVispy.py [hunk 1]
```diff
@@ -14,7 +14,9 @@
 import math
 import random
 import typing
-
+import imageio
```
Comment (sanjayankur31) [inline pyneuroml/plot/PlotMorphologyVispy.py:17]: we need to add this to the `setup.cfg` too, under the `vispy-common` extra

File: pyneuroml/plot/PlotMorphologyVispy.py [hunk 2]
```diff
@@ -238,6 +254,29 @@ def vispy_on_key_press(event):
...
+    def start_recording(canvas, output_file):
+        nonlocal recording, frames
+        recording = True
+        frames = []
+        print(f"Recording started. Output file: {output_file}")
+
+    def stop_recording(canvas):
+        nonlocal recording, frames
+        recording = False
+        output_file = f"output-{len(glob.glob('output-*.mp4'))}.mp4"
```
Comment (sanjayankur31) [inline pyneuroml/plot/PlotMorphologyVispy.py:266]: we generated the name of the output file above already---can we not re-use that here also?

**Review by sanjayankur31 — CHANGES_REQUESTED** [review]
It doesn't currently work for me. Please see the comments (and test it out yourself to make sure it works)

**Pull request closed without merging**